references/cicd-sharedflow-pipeline/.gitignore

```diff
@@ -0,0 +1,4 @@
+target
+node_modules
+.DS_Store
+package-lock.json
```

Comment:
We generally check this in. Rationale: 
https://github.com/npm/cli/blob/latest/docs/content/configuring-npm/package-lock-json.md

`This file is intended to be committed into source repositories [...]`

Reply:
Fixed